src/lpcjobqueue/schedd.py

```diff
@@ -4,7 +4,6 @@
 import re
 
 logger = logging.getLogger(__name__)
-os.environ["CONDOR_CONFIG"] = os.path.join(os.path.dirname(__file__), "condor_config")
 import htcondor  # noqa: E402
```

Comment:
if a script using lpcjobqueue is executed somewhere other than `/srv` then will it still find `/srv/.condor_config` ?

Reply:
Yes, as we export `CONDOR_CONFIG` as a global environment variable in the bootstrap script.